apps/webapp/package.json

```diff
@@ -18,6 +18,7 @@
     "@chakra-ui/react": "^3.24.0",
     "@emotion/react": "^11.14.0",
     "@paddle/paddle-js": "^1.4.0",
+    "@tabler/icons-react": "^3.35.0",
```

Comment:
we use "react-icons" for the tabler icons

you can import them like:

import { TbBrandGoogle, TbBrandGitHub } from 'react-icons/tb';